application/test/process/script_launcher_test.rb

```diff
@@ -54,7 +57,10 @@ class ScriptLauncherTest < ActiveSupport::TestCase
     @download_files_provider.stubs(:pending_files).returns([])
     @upload_files_provider.stubs(:pending_files).returns(['file1'])
 
-    @launcher.expects(:start_process_from_script).with('scripts/launch_detached_process.rb', 'launch_detached_process.log').returns(12345)
+    expected_log_filename = 'launch_detached_process-35-W2025.log'
```

Comment:
I would use the right file name `launch_detached_process-2025-W35.log` for these tests too to avoid confusion